nodejs/package.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "scitt-nodejs-example",
+  "version": "0.0.0",
+  "description": "A nodejs example",
+  "main": "artifact.js",
+  "scripts": {
+    "test": "node artifact.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/scitt-community/scitt-examples.git"
+  },
+  "keywords": [
+    "test"
+  ],
+  "author": "Orie Steele",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/scitt-community/scitt-examples/issues"
+  },
+  "homepage": "https://github.com/scitt-community/scitt-examples#readme"
```

Minor, non-blocking to force updates for anyone who tested previously.

```suggestion
  "homepage": "https://github.com/scitt-community/scitt-examples#readme",
  "dependencies": {
    "@transmute/cli": "^0.8.29"
  }  
```

---

This would be adding the CLI tool as a dependency, currently the example is structured assuming both the transparency tool and the sbom tool are external to the package in question.

---

Sorry I was being too quick and lazy, it should be a dev dependency anyway. I have updated my recommendation accordingly. You would presumably then run it `npm install` in these docs and production "customer builds" would `npm install --omit=dev` like [our beloved StackOverflow guides us with an explanatory answer](https://stackoverflow.com/a/9276112).